gammapy/utils/energy.py

```diff
@@ -189,6 +189,13 @@ def nbins(self):
         return self.size - 1
 
     @property
+    def lin_centers(self):
```

Comment:
You're still calling `lin_centers`.
I've argued before that that's never useful, you'll always get better results by using `log_center`.
Did you want to stick with `lin_centers`? Did I miss your argument why?